docs/source/model-export/export-mnn.rst

```diff
@@ -0,0 +1,27 @@
+.. _icefall_export_to_mnn:
+
+Export to mnn
+==============
+
+We support exporting the following models
+to `mnn <https://github.com/alibaba/MNN>`_:
+
+  - `Zipformer transducer models <https://github.com/k2-fsa/icefall/tree/master/egs/librispeech/ASR/pruned_transducer_stateless7_streaming>`_
+
+We also provide `sherpa-mnn`_
```

Comment:
Please follow
https://github.com/k2-fsa/icefall/blob/eeeeef390b2d7f1aefe742ac069565d5f8eb8a38/docs/source/conf.py#L83

to add a line for `sherpa-mnn` and `MNN`.